### ISSUE TITLE
Scale ruby, iris and jade missing

### ISSUE THREAD

**Issue by silveltman** [endigma/unocss-preset-radix#15]
Don't know if it's just the types, or the package, but I get the following error:

```
Type '"jade"' is not assignable to type '"gray" | "mauve" | "slate" | "sage" | "olive" | "sand" | "tomato" | "red" | "crimson" | "pink" | "plum" | "purple" | "violet" | "indigo" | "blue" | "cyan" | "teal" | "green" | "grass" | ... 8 more ... | "sky"'
```

Get the same error for ruby and iris

**Comment by endigma** [comment]
Are these new colors?

**Comment by silveltman** [comment]
Looking at the commits in the radix colors repo (https://github.com/radix-ui/colors) they seem to have been added about a month ago.

**Comment by endigma** [comment]
Fixed in #16, not released yet.
